packages/shared/lib/assets.ts

```diff
@@ -35,17 +28,25 @@ export const assets = derived(
                 color: '#6E82A4',
             },
         ]
-        if ($assemblyStakingRewards) {
```

Comment:
If we don't use assemblyStakingRewards (or shimmer), then please remove it. Another approach is to create a store that only stores the rewards of the slected account under `stores/participation.ts`, since we won't need a total anymore with the single account refactor. Now we're just leaving dead code in the files.